calls/README.md

```diff
@@ -24,19 +24,19 @@ Below is a list of all prior completed calls and related resources.
 
 | No   | Date       | Time | Agenda  | Video | Notes | Resources |
 | ---- | ---------- | ---- | ------- | ----- | ----- | ---- |
-| 0032 | 2021-01-21 | 1900 UTC | [Agenda](https://github.com/stacksgov/pm/issues/127) | Video TBD | Notes TBD | n/a |
-| 0031 | 2021-01-07 | 1700 UTC | [Agenda](https://github.com/stacksgov/pm/issues/121) | Video TBD | Notes TBD | n/a |
-| 0030 | 2020-12-17 | 1700 UTC | [Agenda](https://github.com/stacksgov/pm/issues/120) | Video TBD | Notes TBD | n/a |
-| 0029 | 2020-12-03 | 1700 UTC | [Agenda](https://github.com/stacksgov/pm/issues/118) | Video TBD | Notes TBD | n/a |
-| 0028 | 2020-11-19 | 1700 UTC | [Agenda](https://github.com/stacksgov/pm/issues/113) | Video TBD | Notes TBD | n/a |
-| 0027 | 2020-11-05 | 1600 UTC | [Agenda](https://github.com/stacksgov/pm/issues/105) | Video TBD | Notes TBD | [healthy ecosystem building from Ryan](https://youtu.be/2Oy9LyVThlg)] |
-| 0026 | 2020-10-22 | 1600 UTC | [Agenda](https://github.com/stacksgov/pm/issues/99) | Video TBD | Notes TBD | n/a |
-| 0025 | 2020-10-08 | 1600 UTC | [Agenda](https://github.com/stacksgov/pm/issues/98) | Video TBD | Notes TBD | [grant proposal](https://paper.dropbox.com/doc/Stacksgov-Grant-Proposal--A9FticUOo4gxWoWRzabNARnkAg-ke9qLunt5n7WwWwZTV56I) |
-| 0024 | 2020-09-23 | 1600 UTC | [Agenda](https://github.com/stacksgov/pm/issues/96) | Video TBD | Notes TBD | n/a |
-| 0023 | 2020-09-10 | 1600 UTC | [Agenda](https://github.com/stacksgov/pm/issues/95) | Video TBD | Notes TBD | n/a |
-| 0022 | 2020-08-20 | 1600 UTC | [Agenda](https://github.com/stacksgov/pm/issues/94) | Video TBD | Notes TBD | n/a |
-| 0021 | 2020-08-06 | 1600 UTC | [Agenda](https://github.com/stacksgov/pm/issues/84) | Video TBD | Notes TBD | [Grant Proposal working doc](https://paper.dropbox.com/doc/Gov-Group-Grant-Proposal--A402pAn8MQVQPIikglkCKxXXAQ-RN1efxJdptsRGVrJIbwou) |
-| 0020 | 2020-07-20 | 1730 UTC | [Agenda](https://github.com/stacksgov/pm/issues/82) | Video TBD | [Notes](https://github.com/stacksgov/pm/issues/82#issuecomment-661339707) | n/a |
+| 0032 | 2021-01-21 | 1900 UTC | [Agenda](https://github.com/stacksgov/pm/issues/127) | [Video](https://zoom.us/rec/share/vmuVAKokl4mkwKUyzxB0_C9g-U9q_RiQlNlbknFjx--YLmJqEFssZM651_8aMGHc.OppKVlyQ2ohxd5O4?startTime=1611256066000) | Notes TBD | n/a |
```

Comment:
I didn't know you could directly post zoom recording links like this - cool! Heads up that zoom periodically auto-deletes old recordings once you've hit your quota, so be careful here.

Reply:
Yes, the goal here is to download each and then put them onto the Youtube channel, hopefully they are all still available! :crossed_fingers: